src/luarocks/static_flags.lua

```diff
@@ -0,0 +1,41 @@
+
+local static_flags = {}
+package.loaded["luarocks.static_flags"] = static_flags
+
+local util = require("luarocks.util")
+local fetch = require("luarocks.fetch")
+local path = require("luarocks.path")
+local search = require("luarocks.search")
+
+util.add_run_function(static_flags)
+
+local function collect_libs(module_name, libs_list)
```

Comment:
Perhaps collect_libs should collect into a table? (instead of generating many strings via appending)